common/src/main/kotlin/streams/utils/Neo4jUtils.kt

```diff
@@ -32,4 +40,61 @@ object Neo4jUtils {
         return db.dependencyResolver
                 .resolveDependency(LogService::class.java)
     }
+
+    fun isEnterpriseEdition(db: GraphDatabaseAPI): Boolean {
+        try {
+            db.execute("CALL dbms.cluster.role()").columnAs<String>("role").next()
```

Comment:
For checking for version, we can use this procedure:

```
CALL dbms.components() YIELD edition
RETURN edition = "enterprise" AS isEnterprise
```

Reply:
Thanks @mneedham I changed the method with your suggestion!